Added User Details Page for Signup

intkhabahmed opened the pull request:
# Fill out the following while publishing a new pull request.

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature


* **What is the current behavior?** (You can also link to an open issue here)
https://github.com/UdacityAndroidDevScholarship/blood-donation/issues/16


* **What is the new behavior (if this is a feature change)?**
Added the user details page to get details from user while registering, changed min-sdk version to 16 and constraint layout to 1.1.0-beta6 for getting some advantage of chaining, added helper strings and vector asset


* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
They might need to change min-sdk version to 16 and constraint layout to 1.1.0-beta6 and rest are same


* **Other information**:
![screenshot_1523647953](https://user-images.githubusercontent.com/18560483/38754654-57e7c75e-3f80-11e8-9188-916b33e5ca4c.png)

ritesh-singh commented:
Great job.
Solve merge conflict, while solving make sure, you are not removing others code.

Also, why constraintlayout and minSdk version is changed.

ritesh-singh commented:
Great job.
Solve merge conflict, while solving make sure, you are not removing others code.

Also, why constraintlayout and minSdk version is changed.

intkhabahmed commented:
I got a merge conflict in Readme file even though i didn't change it. I even solved it but here I don't know why it is showing. Regarding constraint layout version, 1.0.2 did not have chaining and I wanted to use chaining for the first row so I changed it. Also it will provide more features. And regarding min-sdk, I changed it because before 16, we used to define different versions of drawables like mdpi, hdpi, etc. So when I created the vector, it was giving error that it should be defined in other layout versions also. And in constraint layout it was giving cannot resolve error to the id reference of the views when referred in the constraints. That's why I changed it. Also we must keel 16 as previous versions are now having less percentage of users as suggested by Android studio itself.

intkhabahmed commented:
@ritesh-singh I don't have write access, can you see what are conflicts as I have solved it, and .idea folder is also giving conflicts, so can you have a look and let me know so that I can resolve it

ritesh-singh commented:
@intkhabahmed I cannot solve this merge conflict, as github tells me it's too complex to solve on web.
Solve, on local and push.

Make sure, while solving conflict, you are removing others code.

Take pull from upstream develop and solve the conflicts.
